certified-connectors/RAPID Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,645 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "1.15.0",
+		"title": "RAPID Platform (V1)",
```

Comment:
The (V1) won't be necessary in the title since there is already a version tag. Simply `RAPID Platform` is fine :)

Reply:
Updated to drop (V1) label